install/kubernetes/microcks/values.yaml

```diff
@@ -1,4 +1,5 @@
 appName: microcks
+ingress: true
```

Comment:
I think it would be better to name it `ingresses` as it controls the generation of all the `Ingress` objects.